src/Uno.UI.Composition/Composition/Uno/CornerRadius.cs

```diff
@@ -0,0 +1,173 @@
+using System;
+using System.Numerics;
+using Windows.Foundation;
+namespace Uno.UI.Composition;
+
+// A close copy from MUX.CornerRadius
```

Comment:
Are we able to link it instead of cloning it? Lot of logic here, it will be to easy to forget one if we modify the other one.

Reply:
Enabling MUX internals to be visible in Composition is a very slippery slope that I don't want to go down.